suite/tests/api/drdecode_x86.c

```diff
@@ -244,7 +271,7 @@ main()
 
     test_store_source();
 
-    printf("done\n");
+    fprintf(stdout, "done\n");
```

Comment:
Could have remained as it was?  (Aside for future PR's on other tests: Note that tests that interleave output with DR's messages to stderr would want stderr for everything, and they use "print()" from tools.h but this standalone app test does not.)

Reply:
Same reason of https://github.com/DynamoRIO/dynamorio/pull/6512#discussion_r1463943359 .
Should I change it to `print` now that `tools.h` is included? (Or was I not supposed to include `tools.h` here?)

Reply:
If tools.h is successfully included then use print().  There is some build config complexity with tools.h vs config.h vs dr_api.h but if it worked then we're good.

Reply:
Done. The test is built correctly, and the print statements work, at least when I run the `code_api|api.drdecode` test.